_data/ph_authors.yml

```diff
@@ -989,3 +989,19 @@
         fr: |
             Sofia Papastamkou est docteure en histoire et ingénieure d'études en humanités numériques au CNRS. Elle est affectée à la Maison européenne des sciences de l'homme et de la société Lille Nord de France.
         
+  
+  - name: Marie Puren
+  email: marie.puren1406@gmail.com
```

Comment:
Thank you for adding this text @mpuren 👍 Can you please add indentation to every line so that it appears like:

```
- name: Marie Puren
   email: marie.puren1406@gmail.com
   twitter: MariePuren
```

etc... This is a small detail, but it a required format for `.yml` files. If you look at the other bio entries above yours, you can see how the indentation must look.